lab-8 Loops/script.js

```diff
@@ -0,0 +1,31 @@
+for (let num = 1; num < 10; num++) {
+    console.log(num)
+}
+
+for (let num = 10; num > 0; num--) {
+    console.log(num)
+}
+
+let num = 1;
+
+while (num < 10) {
+    console.log(num);
+    num++;
+}
```

Comment:
This isn't a hard and fast rule but I would say that you can group these two pieces of code together into a block, since they are related to the logic
```suggestion
let num = 1;
while (num < 10) {
    console.log(num);
    num++;
}
```